### PR TITLE
Fix sticky header and select the first locale for copy

### PR DESCRIPTION
**Pull request by soupette**
Signed-off-by: soupette <cyril.lpz@gmail.com>

### What does it do?

Fixes the sticky header, and selects the first locale for copy

### CODE REVIEW

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js [hunk 1]
```diff
@@ -17,13 +17,23 @@ import {
 import { getTrad } from '../../utils';
 import { cleanData, generateOptions } from './utils';
 
-const CMEditViewCopyLocale = ({ appLocales, currentLocale, localizations, readPermissions }) => {
+const CMEditViewCopyLocale = props => {
+  if (!props.localizations.length) {
+    return null;
+  }
```
Comment (HichamELBSI) [inline packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:23]: I don't understand why we need to export that condition in another component. 🤔

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js [hunk 2]
```diff
@@ -17,13 +17,23 @@ import {
...
+const Content = ({ appLocales, currentLocale, localizations, readPermissions }) => {
+  const options = generateOptions(appLocales, currentLocale, localizations, readPermissions);
+
   const { formatMessage } = useIntl();
   const dispatch = useDispatch();
   const { allLayoutData, slug } = useContentManagerEditViewDataManager();
   const [isLoading, setIsLoading] = useState(false);
   const [isOpen, setIsOpen] = useState(false);
-  const [value, setValue] = useState(null);
+  const [value, setValue] = useState(options[0]);
```
Comment (HichamELBSI) [inline packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:36]: Do `options[0]` always defined?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:36]: Yes this is why I have split the component into two parts.

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js (hunk 1 above)
Comment (soupette) [reply packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:23]: To make sure we always that the options array contains at least one localisation. See my other comment

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Pull request merged**